### PR TITLE
Adds a human from the past

### PR DESCRIPTION
**Pull request by ahume**

### CODE REVIEW

**Comment by phamann** [comment]
:+1: Ship it!

File: facia/public/humans.txt [hunk 1]
```diff
@@ -137,3 +137,6 @@ From: London, UK
 
 Director of Digital Strategy: Wolfgang Blau
 Twitter: @wblau
+
+Software Architect: Andy Hume
```
Comment (gklopper) [inline facia/public/humans.txt]: `Dodgy fonts hack guy` maybe :wink:

Comment (ahume) [inline facia/public/humans.txt]: Haha, and proud of it.

**Comment by phuu** [comment]
Careful, this guy's a bozo.

**Pull request merged**